CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing
+## Sending a pull request
+Please do things under this text before sending the pull request 
+* Fork
+* Create feature branch
+* Add tests for new feature
+* Make sure Travis CI passes (You can activate Travis CI for your own fork)
+* Make sure coverage does not lower
+* (Merge commits into one)
```

Comment:
The parentheses here were only because I did not know if we want this or not.
I would argue it's a good thing and they should be removed.